src/main/java/pb360/model/embedded/MethodData.java

```diff
@@ -7,7 +7,7 @@
 public class MethodData {
 	private String header;
 	private List<String> body;
-//	private List<String> String;
+
```

Comment:
please always use format commander on the code after changed anything, Ctrl + Shift + F